### PR TITLE
Updates needed for running custom FreeRTOS images

### LINKED ISSUE #1068: [Cyberphys] Package FreeRTOS ECU demo for SRI-Cambridge
The demo should be stand-alone and It should be possible to compile it without FETT-target

### PR DESCRIPTION
**Pull request by podhrmic**
Fixes #1068

### CODE REVIEW

File: besspin/cyberPhys/build.py [hunk 1]
```diff
@@ -108,44 +108,51 @@ def buildInfotainmentServer(tarName, targetId=None):
...
-        logAndExit(f"Failed to populate <besspinFreeRTOSIPConfig.h>.",exc=exc,exitCode=EXIT.Dev_Bug)
-    configHfile.close()
-
-    besspin.target.build.prepareFreeRTOSNetworkParameters(targetId=targetId)
-    besspin.target.build.buildFreeRTOS(targetId=targetId, buildDir=buildDir)
-
-    cp(os.path.join(buildDir,'FreeRTOS.elf'), getSetting('osImageElf',targetId=targetId))
+    if isEnabled('useCustomOsImage',targetId=targetId):
+        targetInfo = f"<target{targetId}>: " if (targetId) else ''
+        printAndLog (f"{targetInfo} Warning, using a custom image")
```
Comment (rtadros125) [inline besspin/cyberPhys/build.py:113]: warnAngLog

File: besspin/cyberPhys/build.py [hunk 2]
```diff
@@ -108,44 +108,51 @@ def buildInfotainmentServer(tarName, targetId=None):
...
+        imgPath = getSetting('pathToCustomOsImage',targetId=targetId)
+        printAndLog (f"{targetInfo} custom image path: {imgPath}")
```
Comment (rtadros125) [inline besspin/cyberPhys/build.py:115]: logging.debug()

File: besspin/cyberPhys/build.py [hunk 3]
```diff
@@ -108,44 +108,51 @@ def buildInfotainmentServer(tarName, targetId=None):
...
+        cp(imgPath, getSetting('osImageElf',targetId=targetId))
+    else:
+        #Netboot on FreeRTOS?
+        if (isEqSetting('elfLoader','netboot',targetId=targetId)):
+            warnAndLog (f"Netboot cannot load FreeRTOS image. Falling to JTAG.", doPrint=False)
+            setSetting('elfLoader','JTAG',targetId=targetId)
+
```
Comment (rtadros125) [inline besspin/cyberPhys/build.py:122]: This part should be before the if (customImage) condition.

File: besspin/cyberPhys/build.py (hunk 1 above)
Comment (podhrmic) [reply besspin/cyberPhys/build.py:113]: Fixed

File: besspin/cyberPhys/build.py (hunk 2 above)
Comment (podhrmic) [reply besspin/cyberPhys/build.py:115]: Fixed

File: besspin/cyberPhys/build.py (hunk 3 above)
Comment (podhrmic) [reply besspin/cyberPhys/build.py:122]: Fixed

**Review by rtadros125 — APPROVED** [review]

**Pull request merged**